plugins/plugin.go

```diff
@@ -16,6 +17,8 @@ const (
 	PluginTypeSink      PluginType = "sink"
 )
 
+type PushFunc func(models.Record)
```

Comment:
@StewartJingga I think this can still be kept. 
```
type Emitter func(models.Record)

Extract(ctx context.Context, emit plugin.Emitter) (err error)

emit()

Reply:
updated to `plugin.Emit` 🙏